rwilliams659/rancid-tomatillos
# Sn feature/comment functionality

#### What changes/additions are a part of this PR?
- Adds functionality for users to be able to view movie comments in the Movie Details section
- Adds functionality for a user to add a new comment if they're logged in
- Adds new components (CommentContainer, Comments, CommentCard) to manage this new functionality
- Implements typechecking with propTypes for new components
- Error handling for sad paths
- Adds testing for for new components
#### Any background context you want to provide?
The CommentContainer is the parent component for the comment functionality
#### Where should the reviewer start?
Start at MovieDetails where the CommentContainer is being rendered.
#### How should this be tested (test suite/manually)?
- Manually test new functionality; in browser select a movie to see the new comment section. When logged in the user should be able to see the comment form and post a new comment. When logged out, the user should be able to see existing comments, but not see the form to add a new comment.
- For the test suite; run 'npm test' in terminal
#### What are the relevant tickets?
Resolves #51 
Resolves #52 
#### Screenshots (if helpful)
Comment form that a user can see when logged in
<img width="1428" alt="Screen Shot 2020-08-31 at 3 06 28 PM" src="https://user-images.githubusercontent.com/62770843/91768882-8dd18c00-eb9b-11ea-87b1-ba873e1857e3.png">
